backend/validators/index.js

```diff
@@ -0,0 +1,5 @@
+const authAPIValidator = require('./auth.api.validator');
+
+module.exports = {
+  authAPIValidator,
```

Comment:
There are a lot of discussions on this topic https://stackoverflow.com/questions/15526107/acronyms-in-camelcase
but I personally prefer `authApiValidator` instead of `authAPIValidator`. For my eyes, it's just easier to read.